JPF/bad_buildings.sql

```diff
@@ -348,9 +421,35 @@ drop table if exists bad_buildings;
 
 create table bad_buildings
```

Comment:
Hot dang this is a long SQL statement. Is there any way to break this down into easier-to-comprehend bits, or does SQL not really allow for that without compromising performance / requiring tons of temporary tables (like the aep yearly tables created directly above)?